brdnet/blood_only_analysis.ipynb

```diff
@@ -0,0 +1,3223 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Blood only analysis\n",
```

Comment:
Any particular reason you decided to use blood data here? (Just curious)

Reply:
I knew from early tests of `find_studies.py` that there would be a large amount of blood data in the dataset (it turns out to be about 10% of the data or ~ 1000 samples). Also since the moonshot goal is to use this as a predictive/diagnostic tool, blood samples are nice as they are much less invasive than most tissue samples.